part1/readme.md

```diff
@@ -0,0 +1,99 @@
+## Installing Stacker
+You can install stacker via pip
+
+```
+pip install stacker stacker_blueprints
```

Comment:
Might be worth pointing out pip's --user flag, i.e. `pip install --user stacker stacker_blueprints`. Requires adding the python local bin path to PATH (e.g. `echo 'export PATH="${HOME}/Library/Python/2.7/bin:${PATH}"' >> ${HOME}/.${SHELL_NAME}_profile` on macOS), but has the major advantage of not needing homebrew python / sudo / etc